models/user.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"github.com/beego/beego/v2/client/orm"
+)
+
+type User struct {
+	Default
+
+	Email             string `orm:"unique;size(128)"`
+	EncryptedPassword string
+}
+
+func init() {
+	orm.RegisterModel(new(User))
+
+}
+
+// AddUser insert a new User into database and returns last inserted Id on success.
+func AddUser(m *User) (id int64, err error) {
+	ormer := orm.NewOrm()
```

Comment:
@gutakk Seems like we cannot move this to a global variable and assign the value on `init()` because `orm.NewOrm()` would bootstrap all models and try to connect to the database but on `init()` the database is not registered yet.
![Screen Shot 2564-01-19 at 17 40 20](https://user-images.githubusercontent.com/1772999/105023681-d8d23a00-5a7d-11eb-873c-b61cc89b02ac.png)

so it would cause a run time error
![Screen Shot 2564-01-19 at 17 27 47](https://user-images.githubusercontent.com/1772999/105022210-1209aa80-5a7c-11eb-90f4-f0796bfafcc3.png)